master/buildbot/reporters/words.py

```diff
@@ -121,207 +96,91 @@ def parseArgs(self, *args):
             self['reason'] = " ".join(args)
 
 
-class Contact(service.AsyncService):
-
-    """I hold the state for a single user's interaction with the buildbot.
-
-    There will be one instance of me for each user who interacts personally
-    with the buildbot. There will be an additional instance for each
-    'broadcast contact' (chat rooms, IRC channels as a whole).
+dangerous_commands = []
+
+
+def dangerousCommand(method):
+    command = method.__name__
+    if not command.startswith('command_'):
+        raise ValueError('@dangerousCommand can be used only for commands')
+    dangerous_commands.append(command[8:])
+    return method
+
+
+def convertTime(seconds):
```

Comment:
This could be moved to the `buildbot.util` module as it's likely to be useful in other places of Buildbot. Maybe for a future PR.

Reply:
I think it is specific to `words` and commands defined here (and below).

I would rather move `overrideCommand` to `buildbot.util`, as it can have more uses (it copies docstring and `usage` attr of the base method).

Reply:
Ok, I did not realize you talked about `convertTime`. I moved it to `buildbot.util`.